src/sentry/models/groupshare.py

```diff
@@ -0,0 +1,38 @@
+"""
+sentry.models.groupshare
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+:copyright: (c) 2010-2014 by the Sentry Team, see AUTHORS for more details.
+:license: BSD, see LICENSE for more details.
+"""
+from __future__ import absolute_import
+
+from uuid import uuid4
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey, Model, BaseManager, sane_repr
+
+
+class GroupShare(Model):
+    """
+    A Group that was shared publicly.
+    """
+    __core__ = False
+
+    project = FlexibleForeignKey('sentry.Project')
+    group = FlexibleForeignKey('sentry.Group', unique=True)
+    guid = models.CharField(max_length=32, unique=True, default=lambda: uuid4().hex)
```

Comment:
can we call this uuid for consistency with the rest of the codebase that already uses uuid? (eg: dsyms, symcaches, etc.)

guid is a microsoft term that is used for a subset of uuids.

Reply:
We definitely use `guid` in our codebase too inconsistently. `ScheduledDeletion` for example and within `deletions` code we use guid as well as new analytics stuff.

So I agree, but I think we're just inconsistent on both sides and don't think it's worth picking a side here unless we were to change everything.

Reply:
So looking, I see exactly 1 model with a `uuid` column, and exactly 1 with `guid`. :)

But I didn't know that GUID was specifically a Microsoft term. So I'll change this to uuid so I feel better about it.

Reply:
Done e53ba536ee598da49b13d1812bc6f8c2235d6312